listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx

```diff
@@ -373,10 +360,23 @@ export default class UserEntityChart extends React.Component<
     window.history.pushState(
       null,
       "",
-      `?page=${page}&range=${range}&entity=${entity}`
+      this.buildURLParams(page, range, entity)
```

Comment:
There is one more instance or the template string at line 81 in `componentDidMount` that should be changed

Reply:
thanks!